src/V8JsiRuntime.cpp

```diff
@@ -581,10 +582,12 @@ V8Runtime::V8Runtime(V8RuntimeArgs &&args) : args_(std::move(args)) {
 }
 
 V8Runtime::~V8Runtime() {
+  // TODO: add check that destruction happens on the same thread id as construction

```

Comment:
>// TODO: add check that destruction happens on the same thread id as construction [](start = 2, length = 81)

This is what RN/Hermes does : [https://github.com/facebook/react-native/blob/42c1957aff726b7f0a29eb26e81b69bb50bc4710/ReactAndroid/src/main/java/com/facebook/hermes/reactexecutor/HermesExecutorFactory.cpp#L80](https://github.com/facebook/react-native/blob/42c1957aff726b7f0a29eb26e81b69bb50bc4710/ReactAndroid/src/main/java/com/facebook/hermes/reactexecutor/HermesExecutorFactory.cpp#L80)